components/library/libpng12/libpng12.p5m

```diff
@@ -22,33 +22,25 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-# Commented out files should be delivered by the most recent
-# libpng version (currently it is libpng 1.4)
+# Certain files should be delivered by the most recent
```

Comment:
Leave original comment and avoid removing commented out files - this is a kind of documentation